middleware/cache/README.md

```diff
@@ -91,6 +98,7 @@ type Config struct {
 var ConfigDefault = Config{
 	Next:         nil,
 	Expiration:   1 * time.Minute,
+	CacheHeader:  "S-Cache",
```

Comment:
The default header should be X-Cache, it's used by many other CDN's and X indicates a non standard header